ludwig/backend/ray.py

```diff
@@ -187,25 +167,30 @@ def train_fn(
         initialize_pytorch(horovod=hvd)
 
         train_shard = RayDatasetShard(
-            rt.get_dataset_shard("train"),
+            session.get_dataset_shard("train"),
             features,
             training_set_metadata,
+            data_loader_kwargs,
         )
 
         try:
-            val_shard = rt.get_dataset_shard("val")
+            val_shard = session.get_dataset_shard("val")
         except KeyError:
             val_shard = None
 
+        eval_data_loader_kwargs = copy.deepcopy(data_loader_kwargs)
+        eval_data_loader_kwargs["shuffle"] = False
```

Comment:
Don't make this assumption, but pass this in first and overwrite with data_loader_kwargs so that incase it isn't specified, only in those scenarios we fall back to `shuffle = False`